docs/logic-simple-expr.md

```diff
@@ -24,7 +24,7 @@ Every Ergo expression can be _evaluated_, which means it should compute some val
 
 ## Operators
 
-You can apply operators to values. Those can be used for arithmetics, to compare two values, to concatenate two string values, etc.
+You can apply operators to values. Those can be used for arithmetic operations, to compare two values, to concatenate two string values, etc.
```

Comment:
@jeromesimeon flagging this because unsure if it makes more sense.

The rest of the changes are great though!